samples/force-sensitive-resistor/FsrWithAdcSample.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using Iot.Device.Mcp3008;
+
+namespace force_sensitive_resistor
+{
+    class FsrWithAdcSample
+    {
+        private int _pinNumber = 0; // pin number to which FSR output connected
+        private int _resistance { get; set; } = 10_000; // kOhm
+        private int _voltageSupplied { get; set; } = 3_300;  // 3300mV = 3.3V
+
+
+        private int CalculateVoltage(int readValue)
+        {
+            // This sample used Mcp3008 ADC which analog voltage read output ranges from 0 to 1023 (10 bit) 
+            // mapping it to corresponding milli voltage, update output range if you use different ADC
+            return _voltageSupplied * readValue / 1023;
+        }
+
+        private int CalculateFsrResistance(int fsrVoltage)
+        {
+            // Formula: FSR = ((Vcc - V) * R) / V
+            if (fsrVoltage > 0)
+            {
+                return (_voltageSupplied - fsrVoltage) * _resistance / fsrVoltage;
+            }
+            return 0;
+        }
+
+
+        private int CalculateForce(int resistance)
+        {
+            if (resistance > 0)
+            {
+                int force;
+                int fsrConductance = 1_000_000 / resistance; // in micro ohms
```

Comment:
the units don't match - if you divide by resistance which is already in ohms then 1M would have to be in ohms squared which doesn't make much sense

Reply:
Yeah it is confusing, conductance = 1/R i.e. inverse of resistance, here 2 calculations made, getting inverse of resistance and converting into micro ohms (1/μΩ), updated comment